app/models/spree/display.rb

```diff
@@ -4,6 +4,7 @@ class Display < ActiveRecord::Base
     has_many :products, through: :display_product, class_name: 'Spree::Product'
 
     before_validation :set_default_screen_size
+    before_validation :set_default_active, on: :create
```

Comment:
yeha bata garnu bhanda default status database ma nai rakhau

Reply:
yeha bata ni rakheko chu

<img width="659" alt="image" src="https://github.com/user-attachments/assets/6d338672-ed9a-44de-b876-25a19b7b92b6">

Reply:
then we dont need it in code..
we can just rely on db..